python/lsst/obs/base/ingest_tests.py

```diff
@@ -245,13 +244,15 @@ def testWriteCuratedCalibrations(self):
 
         self._writeCuratedCalibrations()
 
-        dataId = {"instrument": self.instrumentName}
-        butler = Butler(self.root, run=self.outputRun)
+        butler = Butler(self.root, writeable=False)
         for datasetTypeName in self.curatedCalibrationDatasetTypes:
-            with self.subTest(dtype=datasetTypeName, dataId=dataId):
```

Comment:
Probably should keep the subTest here so we can tell which dataset type is broken.

Reply:
The dataset type name is already in the assert message below.

Reply:
although subTest runs the full loop whereas a failure in the first item in the loop stops the entire test.

Reply:
Yeah, which I usually hate, but it seems safe here.  I'll put it back in.